apps/wallet/src/manifest/manifest.json

```diff
@@ -10,7 +10,12 @@
     "action": {
         "default_popup": "ui.html?type=popup"
     },
-    "host_permissions": ["https://api.devnet.iota.cafe/", "https://api.testnet.iota.cafe/"],
+    "host_permissions": [
+        "http://localhost:9000/",
+        "http://localhost:9124/",
+        "http://*/*",
+        "https://*/*"
```

Comment:
This allows access to any website over HTTP and HTTPS. Isn't it too much of a risk for the wallet?
I think it is safer to specify the domains that can access the wallet, instead of allowing access to any domain

Reply:
users can use any node freely, therefore this makes perfect sense. Metamask does the same thing